Source/CharismaModule/Private/CharismaInitialization.cpp

```diff
@@ -0,0 +1,44 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "CharismaInitialization.h"
+
+UCharismaInitialization::UCharismaInitialization(const FObjectInitializer& ObjectInitializer) : Super(ObjectInitializer), WorldContextObject(nullptr)
+{
+}
+
+UCharismaInitialization* UCharismaInitialization::CharismaInitialization(UObject* WorldContextObject, const int32 StoryId, const int32 StoryVersion, const FString& ApiKey)
+{
+
+	UCharismaInitialization* BlueprintNode = NewObject<UCharismaInitialization>();
+	BlueprintNode->WorldContextObject = WorldContextObject;
+	BlueprintNode->InitStoryId = StoryId;
+	BlueprintNode->InitStoryVersion = StoryVersion;
+	BlueprintNode->InitApiKey = ApiKey;
+	return BlueprintNode;
+}
+
+void UCharismaInitialization::Activate()
+{
+
+	UPlaythrough* RefPlaythrough;
+
+	UCharismaAPI::CreateCharismaPlaythroughObject(RefPlaythrough);
+
+	RefPlaythrough->OnTokenCreated.AddDynamic(this, &UCharismaInitialization::BroadcastSuccess);
+	RefPlaythrough->OnTokenFailCreation.AddDynamic(this, &UCharismaInitialization::BroadcastFailure);
```

Comment:
Really minor, but to be consistent with Blueprint `BroadcastSuccess` and `BroadcastFailure`, perhaps we can use `OnTokenCreationSuccess` and `OnTokenCreationFailure`?